pkg/templates/antiaffinity/template.go

```diff
@@ -28,9 +29,10 @@ func defaultTopologyKeyMatcher(topologyKey string) bool {
 
 func init() {
 	templates.Register(check.Template{
-		HumanName:   "Anti affinity not specified",
-		Key:         templateKey,
-		Description: "Flag objects with multiple replicas but inter-pod anti affinity not specified in the pod template spec",
+		HumanName: "Anti affinity not specified",
+		Key:       templateKey,
+		Description: "Flag objects with multiple replicas but inter-pod anti affinity not specified in the pod " +
+			"template spec",
```

Comment:
Why this change?

Reply:
Refactored to not exceed line-length of 130 characters.